shadowrunners/macgee.md

```diff
@@ -0,0 +1,194 @@
+* **Physical:** 10
+* **Stun:** 11
+* **Overflow:** 3
+* **Recoil Compensation:** 2 (1 + Str/3 -- p. 175)
+* **Defense:**	7 (Rea + Int)
+* **Armor:** 11		
+* **Dmg Resistance:** 14 (Body + Armor)
+
+##Augmentations
+
+Nope. Uh-uh. Nadda.
+
+##Melee Weapons
+
+Hahaha that's funny.
+
+##Ranged Weapons
+
+| Qty | Weapon | Damage | Accuracy | AP | Mode | RC | Ammo | Notes |
+| --: | --- | --- | --- | --- | --- | --- | --- | --- |
+|   2 |Ruger Super Warhawk | 9P | 5(7) | -2 | SS | - | 6 (cy) | Smartlink [integral], Hidden Arm Slide (-1 Concealability, -1 Quick Draw) |
+
+##Armor
+
+**Total**: 11
+
+| Armor | Rating | Notes |
+| --- | --- | --- |
+| Lined Coat | 9 | -2 Concealability |
+| Classy Wide-Brimmed Hat (Helmet) | +2 | Trodes (4 spare capacity), Cowboy Hat Customization (5¥, GM Option) |
+
+##Qualities
+
+| Quality | Notes |
+| --- | --- |
+| SINner (Corporate SIN) | Father is CEO of an AA corp specializing in tech. Refused to get mods like family wanted in favor of studying magic. Kicked out / walked out after a big fight. |
+| Mentor Spirit | Raven. +2 to Con tests and Manipulation spells. Charisma + Willpower (3) Test to avoid exploiting someone else’s misfortune to your own advantage or to pull a clever trick or prank even if it’s to the disadvantage of your friends. |
+| Quick Healer | +2 dice pool modifer to all Healing Tests made on/for/by her, including magical healing |
+| Ambidextrous | Can handle objects equally well with either hand, including shooting |
+
+## Skills
+
+**Active**
+
+| Skill | Rating | Atribute |
+| --- | --- | --- |
+| Acting Skill Group | 1 | Cha |
+| Influence Skill Group | 1 | Cha |
+| Intimidation | 1 | Cha |
+| Banishing | 1 | Mag |
+| Summoning | 1 | Mag |
+| Disenchanting | 1 | Mag |
+| Spellcasting | 6 | Mag |
+| Counterspelling | 5 | Mag |
+| Pistols | 6 | Agi |
+| Palming | 5 | Agi |
+| Sneaking | 2 | Agi |
+| Disguise | 2 | Int |
+| Perception | 2 | Int | 
+| Assensing | 2 | Int | 
+| Astral Combat | 1 | Wis |
+
+**Specializations **
+
+| Specialization | Skill | 
+| --- | --- | 
+| Revolvers | Pistols |
+| Illusion Spells | Spellcasting |
+| Legerdemain | Palming |
+
+**Knowledge**
+
+| Skill                | Rating | Category | Attribute |
+| -------------------- | ------ | -------- | --------- |
+| English              | N      | ...      |  ...      |
+| Gaelic               | 2      | ...      |  ...      |
+| Corporations         | 2      | Pro      | Int       |
+| Politics             | 1      | Pro      | Int       |
+| Shadowrunning Scene  | 1      | Pro      | Int       |
+| High Society         | 1      | Pro      | Int       |
+| Magical Suppliers    | 2      | Interest | Int       |
+| Magical Groups       | 2      | Interest | Int       |
+| Magical Theory       | 1      | Interest | Int       |
+| Magical Lore         | 2      | Interest | Int       |
+
+##Contacts
+
+| Name | Loyalty | Connection | Favor |
+| --- | --- | --- | --- | 
+| Talismonger | 3 | 4 | n/a |
```

Comment:
The name of your Talismonger is Margie Dancing Crow.  She resides in and around the Renton/Salish-Shidhe Council border area.  Rumor has it that she knows both the fastest, and the quickest ways to cross that boundary without questions...